tips/TIP-0027/irc27.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  // name of the standard
+  "standard" : "IRC27",
+  // type can be "image", "audio", "video", "document", "3dAsset"
+  "type": "image",
```

Comment:
why wouldn't you just consider standard MIME types? (https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types)